log/logrus_writer_test.go

```diff
@@ -207,6 +208,123 @@ func (s *LogrusTestSuite) TestLogrus() {
 				assert.True(s.T(), file.Contain(dailyLog, "test.panic: Goravel: World"))
 			},
 		},
+		{
+			name: "Code",
+			setup: func() {
+				mockDriverConfig(mockConfig)
+
+				log = NewLogrusApplication(mockConfig)
+				log.Code("code").Info("Goravel")
+			},
+			assert: func() {
+				assert.True(s.T(), file.Contain(singleLog, "test.info: Goravel\ncode: \"code\""))
+				assert.True(s.T(), file.Contain(dailyLog, "test.info: Goravel\ncode: \"code\""))
+			},
+		},
+		{
+			name: "Hint",
+			setup: func() {
+				mockDriverConfig(mockConfig)
+
+				log = NewLogrusApplication(mockConfig)
+				log.Hint("hint").Info("Goravel")
+			},
+			assert: func() {
+				assert.True(s.T(), file.Contain(singleLog, "test.info: Goravel\nhint: \"hint\""))
+				assert.True(s.T(), file.Contain(dailyLog, "test.info: Goravel\nhint: \"hint\""))
+			},
+		},
+		{
+			name: "In",
+			setup: func() {
+				mockDriverConfig(mockConfig)
+
+				log = NewLogrusApplication(mockConfig)
+				log.In("domain").Info("Goravel")
+			},
+			assert: func() {
+				assert.True(s.T(), file.Contain(singleLog, "test.info: Goravel\ndomain: \"domain\""))
+				assert.True(s.T(), file.Contain(dailyLog, "test.info: Goravel\ndomain: \"domain\""))
+			},
+		},
+		{
+			name: "Owner",
+			setup: func() {
+				mockDriverConfig(mockConfig)
+
+				log = NewLogrusApplication(mockConfig)
+				log.Owner("team@goravel.dev").Info("Goravel")
+			},
+			assert: func() {
+				assert.True(s.T(), file.Contain(singleLog, "test.info: Goravel\nowner: \"team@goravel.dev\""))
+				assert.True(s.T(), file.Contain(dailyLog, "test.info: Goravel\nowner: \"team@goravel.dev\""))
+			},
+		},
+		{
+			name: "Request",
+			setup: func() {
+				mockDriverConfig(mockConfig)
+
+				log = NewLogrusApplication(mockConfig)
+				log.Request(nil).Info("Goravel")
```

Comment:
S: Create a TestRequest to test this feature, like: https://github.com/goravel/framework/blob/c38f2bf119f3858ab9b64b20a4199e6f67ca3161/http/middleware/throttle_test.go#L282